### DISCUSSION TITLE
Get synpress to run on Eth mainnet

### DISCUSSION THREAD

**Discussion by crashnaut** [Synthetixio/synpress#373] [Q&A] [answered]
I have looked through the synpress readme, and the kwenta repos.

The default seems to be the kovan test network. Is there a way to connect simply to Eth Mainnet?

**Comment by drptbl** [comment, marked as answer]
@LorenzoDiRosso Yes, of course! Just run your e2e tests with following environmental variable set:
`NETWORK_NAME=mainnet`.

You can find it in the docs:
https://github.com/Synthetixio/synpress#-important
<img width="775" alt="Screenshot 2022-04-06 at 22 27 23" src="https://user-images.githubusercontent.com/8177587/162064881-f5f9bb90-0574-487d-9ca6-265f538ee84c.png">
